def error(msg):
    exit(f'ERROR: {msg}')
